src/Managed.sol

```diff
@@ -3,23 +3,49 @@ pragma solidity ^0.8.17;
 
 import {Upgradeable} from "./Upgradeable.sol";
 import {StorageSlot} from "openzeppelin-contracts/utils/StorageSlot.sol";
+import {EnumerableSet} from "openzeppelin-contracts/utils/structs/EnumerableSet.sol";
+
+using EnumerableSet for EnumerableSet.AddressSet;
 
 /// @notice A mix-in for contract pauseability and upgradeability.
 /// It can't be used directly, only via a proxy. It uses the upgrade-safe ERC-1967 storage scheme.
 ///
 /// All instances of the contracts are paused and can't be unpaused.
 /// When a proxy uses such contract via delegation, it's initially unpaused.
 abstract contract Managed is Upgradeable {
-    /// @notice The pointer to the storage slot with the boolean holding the paused state.
-    bytes32 private immutable pausedSlot = erc1967Slot("eip1967.managed.paused");
+    /// @notice The pointer to the storage slot holding a single `ManagedStorage` structure.
+    bytes32 private immutable _storageSlot = erc1967Slot("eip1967.managed.storage");
```

Comment:
```suggestion
    bytes32 private immutable _storageSlot = _erc1967Slot("eip1967.managed.storage");
```
I just realized this, if we want to be consistent we should change it.

Reply:
Good catch, will fix it in an upcoming PR!